src/topp/FileFilter.cpp

```diff
@@ -306,6 +307,9 @@ class TOPPFileFilter :
     registerStringOption_("int", "[min]:[max]", ":", "Intensity range to extract", false);
 
     registerFlag_("sort", "Sorts the output according to RT and m/z.");
+    registerStringList_("remove_meta", "<name> 'lt|eq|gt' <value>", StringList(), "Expects a 3-tuple (=3 entries in the list), i.e. <name> 'lt|eq|gt' <value>; the first is the name of meta value, followed by the comparison operator (equal, less or greater) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)!", false);
+    registerFlag_("remove_all_meta","Remove ALL metadata. This happens after filtering by remove_meta");
```

Comment:
misleading name IMHO.
This flag removed metadata ARRAYS from MSExperiments and has nothing to do with metadata as userparams.

Reply:
I'm not especially attached to the name. Do you have a suggestion that'd fit better?

Reply:
Also it simply calls `clearMetaInfo` on the maps, which only removes meta info that is directly related to the Feature/ConsensusMap itself but not inside the Features. I dont think that is really what I expect (in the case of mzML it pertains to removing actual data that is `not` mz or intensity, in the case of Feature/Consensus Maps it pertains to removing meta data associated with the while file and not with features). I think this is overall quite confusing.